velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -140,6 +142,8 @@ class NthValueTest : public WindowTestBase {
     }
   }
 
+  FuzzerGenerator rng_;
```

Comment:
You need to initialize rng_ with a seed in the class constructor. Otherwise, it will be seeded with the same default seed at every run (and hence generate the same random numbers). The initialization can be something like `rng_.seed(std::time(nullptr))`.

Reply:
Makes sense. Done. Thanks.